test-kit/components/portal-driver.ts

```diff
@@ -0,0 +1,21 @@
+import * as ReactDOM from 'react-dom';
+import {Portal} from '../../src';
+import {DriverBase} from 'test-drive-react';
+
+export class PortalTestDriver extends DriverBase {
+    public static ComponentClass = Portal;
+    private ref: Portal;
+
+    constructor(public readonly instance: Portal) {
+        super(() => ReactDOM.findDOMNode(instance));
+        this.ref = instance;
+    }
+
+    public get portal() {
```

Comment:
does this represent the root? if so call this `root`.

Reply:
yes